tests/api_resources/abstract/test_custom_method.py

```diff
@@ -121,26 +99,28 @@ def test_call_custom_list_method_class_paginates(self, request_mock):
 
         # Explicitly assert that the pagination parameter were kept for the
         # second request along with the starting_after param.
-        request_mock.assert_requested(
+        http_client_mock.assert_requested(
             "get",
-            "/v1/myresources/mid/do_the_list_thing",
-            {"starting_after": "cus_2", "param1": "abc", "param2": "def"},
+            path="/v1/myresources/mid/do_the_list_thing",
+            query_string="param1=abc&param2=def&starting_after=cus_2",
         )
 
         assert ids == ["cus_1", "cus_2", "cus_3"]
 
-    def test_call_custom_stream_method_class(self, request_mock):
-        request_mock.stub_request_stream(
+    def test_call_custom_stream_method_class(self, http_client_mock_streaming):
+        http_client_mock_streaming.stub_request(
             "post",
-            "/v1/myresources/mid/do_the_stream_thing",
-            "response body",
+            path="/v1/myresources/mid/do_the_stream_thing",
+            rbody=util.io.BytesIO(str.encode("response body")),
```

Comment:
Why is this needed?

Reply:
`request_mock` would [convert the string to bytes internally](https://github.com/stripe/stripe-python/blob/3a3454408974a76c7dcbcc1a63951793142b3d8e/tests/request_mock.py#L202) when stubbing the request. I pulled that logic out so that the mock HTTP client directly returns bytes.